docs/TestCoverage.md

```diff
@@ -7,7 +7,7 @@
 ## Summary
 Node tests have covered 114/121 (94.21%, 5 generators excluded) common operators.
 
-Node tests have covered 0/4 (0.00%, 0 generators excluded) experimental operators.
+Node tests have covered 0/0 (N/A) experimental operators.
```

Comment:
Minor suggestion - maybe remove this line altogether since there are no experimental ops now.

Reply:
I will create a separate pr to fix it.